src/App.tsx

```diff
@@ -1,26 +1,228 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useEffect, useState, useRef } from 'react';
+import { USER_ID } from './api/todos';
+import { Todo } from './types/Todo';
+import * as postService from './api/todos';
+import classNames from 'classnames';
+import { Filter } from './types/Filter';
+import { Header } from './components/Header';
+import { TempToDo } from './components/TempToDo';
+import { Footer } from './components/Footer';
+import { ToDoItem } from './components/ToDoItem';
 
 export const App: React.FC = () => {
+  // #region loadToDOs
+  const [todos, setToDos] = useState<Todo[]>([]);
+  const [, setLoading] = useState<boolean>(false);
+  const [loadingTodoIds, setloadingTodoIds] = useState<number[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [filter, setFilter] = useState<Filter>(Filter.All);
+  const [tempToDo, setTempToDo] = useState<Todo | null>(null);
+
+  const inputRef = useRef<HTMLInputElement>(null);
+
+  const loadToDos = () => {
```

Comment:
In fact,  once you remove `setLoading` from this function you can entirely remove the function and just do this inside of useEffect callback:

```
postService
      .getTodos(USER_ID)
      .then(setToDos)
      .catch(() => setErrorMessage('Unable to load todos'))
      .finally(() => setLoading(false));
```